File: app/src/main/java/com/rtukpe/fixtures/ui/competition/table/TableMvpView.java
package com.rtukpe.fixtures.ui.competition.table;

import com.rtukpe.fixtures.data.model.LeagueTable;
import com.rtukpe.fixtures.ui.base.MvpView;

import java.util.ArrayList;

public interface TableMvpView extends MvpView {

    void updateTableStandings(ArrayList<LeagueTable.Standing> standings);

}

File: app/src/main/java/com/rtukpe/fixtures/ui/competition/table/TableMvpView.kt
package com.rtukpe.fixtures.ui.competition.table

import com.rtukpe.fixtures.data.model.LeagueTable
import com.rtukpe.fixtures.ui.base.MvpView
import java.util.*

interface TableMvpView : MvpView {

    fun updateTableStandings(standings: ArrayList<LeagueTable.Standing>)

}
